albam/engines/mtfw/texture.py

```diff
@@ -228,11 +236,14 @@ def _find_texture_index(mtfw_material, texture_type, from_mrl=False):
 
             if TEX_TYPE_MAPPER.get((shader_object_id >> 12)) == texture_type:
                 tex_index = resource.value_cmd.tex_idx
+                if texture_type.value == 5:
+                    shader_objects = get_shader_objects()
```

Comment:
I think I know what you tried to do here, we can actually get the string value and covert the TODO below. I'll post it when I get some time.